token-group/example/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! Crate defining an example program for creating SPL token groups
+//! using the SPL Token Group interface.
+
+#![deny(missing_docs)]
+#![cfg_attr(not(test), forbid(unsafe_code))]
```

Comment:
micro-nit: I know we have this just about everywhere else, but we can forbid unsafe code all the time!
```suggestion
#![forbid(unsafe_code)]
```